build.xml

```diff
@@ -221,7 +214,15 @@
       <arg value="--always"/>
     </exec>
     <!-- Set version if git describe failed. -->
-    <property name="adaptorlib.version" value="unknown"/>
+    <property name="adaptorlib.version" value="unknown-${DSTAMP}"/>
```

Comment:
so before and after:
if explciit version is given then that is used
if git describe gives version then that is used

however now if no version is given then version is "unknown-$time" where before it was just "unknown".

that seems ok.